### ISSUE TITLE
TIC-80 keeps stopping (Android)

### ISSUE THREAD

**Issue by stevenfelix505** [nesbox/TIC-80#1499]
TIC-80 keeps stopping when i try to launch it.
Version : 0.90.1706

Screenshots :
![Screenshot_20210719-195518_Pixel Launcher.jpg](https://user-images.githubusercontent.com/57618009/126164944-b323677c-bc89-439e-aa80-d8d9cdd9a3a5.jpg)

In previous version (0.80.1344), it works flawlessly without this issue.

~~Reuploading this screenshot coz the first one didn't show the image attachment. And currently i'm using the 0.80.1344~~

**Comment by nesbox** [comment]
Can you please delete TIC-80 and completely reinstall it.
Thanks.

**Comment by stevenfelix505** [comment]
Same as before
![Screenshot_20210719-214857_Pixel Launcher.jpg](https://user-images.githubusercontent.com/57618009/126180054-2f24e82b-73ec-4da6-af7c-df0e14480af4.jpg)

**Comment by nesbox** [comment]
Ok, could you please check this build (with enabled SDLGPU lib)?
https://github.com/nesbox/TIC-80/actions/runs/1048737701
[tic80-android.zip](https://github.com/nesbox/TIC-80/files/6848177/tic80-android.zip)
Thank you.

**Comment by stevenfelix505** [comment]
Still the same as before
![Screenshot_20210720-201700_Pixel Launcher.jpg](https://user-images.githubusercontent.com/57618009/126331253-b65db6f5-961c-4e86-9d20-d818dc43a810.jpg)

**Comment by nesbox** [comment]
What version of Android? Also what device you are using?

**Comment by stevenfelix505** [comment]
OS : Android 10
Device : Samsung SM-A105G

**Comment by nesbox** [comment]
Can't find any device to reproduce :(
It works everywhere for me...

**Comment by ChildishGiant** [comment]
I can't replicate this on Android 11 and a oneplus 5t

**Comment by EODROID** [comment]
Ok,it's fine 😊 i gonna wait for next update... And maybe that's work...

**Comment by SergioFLS** [comment]
happening to me as well on a Moto E4
![Screenshot_20210720-224927](https://user-images.githubusercontent.com/34007063/126423361-ac8df99c-0929-4178-9435-8199273bae9b.png)
all i've done is just open the app.

**Comment by EODROID** [comment]
@SergioFLS is that's you in the discord?

**Comment by nesbox** [comment]
ok, the problem is somewhere here, the config initialization part is crashing...
```
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
pid: 0, tid: 0 >>> com.nesbox.tic <<<

backtrace:
  #00  pc 00000000000dbbc6  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/lib/arm/libtic80.so (tic_cart_load+61)
  #00  pc 0000000000044029  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/lib/arm/libtic80.so
  #00  pc 0000000000044007  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/lib/arm/libtic80.so
  #00  pc 0000000000043f09  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/lib/arm/libtic80.so (initConfig+56)
  #00  pc 0000000000043157  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/lib/arm/libtic80.so (studioInit+606)
  #00  pc 000000000004044f  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/lib/arm/libtic80.so (SDL_main+94)
  #00  pc 000000000008c8c3  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/lib/arm/libtic80.so (Java_org_libsdl_app_SDLActivity_nativeRunMain+398)
  #00  pc 00000000000027c9  /data/app/com.nesbox.tic-Ib5bZcQL1jpjRV1s-ISY0w==/oat/arm/base.odex (offset 0x2000)
```

**Comment by nesbox** [comment]
Finally, found a device with the crash, investigating...

**Comment by nesbox** [comment]
fixed here 64baf44

**Comment by stevenfelix505** [comment]
Thank you👍
![Screenshot_20210721-230050_TIC-80](https://user-images.githubusercontent.com/57618009/126522196-82e50531-18ae-432d-928d-c2b1b8d48254.jpg)
